### PR TITLE
Fix handler mask trigger GPU platform by default

### PR DESCRIPTION
**Pull request by fehiepsi**
We should not set the default arguments to jax.numpy device array. Otherwise, we can't change the default platform (GPU).

### CODE REVIEW

**Comment by neerajprad** [comment]
Could you expand on what you meant by "we should not set the default arguments to jax.numpy device array"?

**Comment by fehiepsi** [comment]
If we have some `jax.numpy` code globally, GPU will be set and we can't change to use CPU by default. I faced this issue while running test code for SA sampler (pytest runs in GPU instead of CPU).
